apps/web-app/server/api/auth/status.get.ts

```diff
@@ -0,0 +1,7 @@
+type ResponseType = 'loading' | 'authenticated' | 'unauthenticated' | 'error';
+export default eventHandler((event) => {
+  if (event.context.user.id) {
+    return { status: 'authenticated' as ResponseType };
+  }
+  return { status: 'unauthenticated' as ResponseType };
+});
```

Comment:
Cant we just type the eventhandler? So we make sure we can only return those values?

Reply:
👀 looking into it